scripts/verify-test-resource.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -o nounset # don't allow use of undefined variables
+set -o pipefail # if any single command in a pipe chain exits non-zero, set the exit status to that
+set -o errexit # if any single command exits non-zero, exit the entire script with that status
```

Comment:
@slarse Had one confusion here. My script **does not** exit when at least one resource fails to compile. Although this is desired, the script should exit with a non-zero code, right?

Reply:
I'm not sure what you mean by the script not exiting. Do you mean that it doesn't immediately exit when the first resource fails to compile? Then that's because you've got it in an `if [[ ]]`, the exit status is "consumed" the conditional.

Which is what you wanted? So you can report all failed files at the end of the script?

Reply:
> Do you mean that it doesn't immediately exit when the first resource fails to compile?

I meant exactly this. :)

> Then that's because you've got it in an if [[ ]], the exit status is "consumed" the conditional.

Got it.

> Which is what you wanted?

Yes, that's exactly what I wanted but I thought I would have to write some extra code for it but it just worked out for me.

Reply:
Okay, then all is good :)